packages/babel-traverse/test/arrow-transform.js

```diff
@@ -65,7 +65,7 @@ describe("arrow function conversion", () => {
       _supercall();
       () => _supercall();
     `,
-      { methodName: "constructor" },
+      { methodName: "constructor", extend: true },
```

Comment:
I think `extend: true` was missing here and not left out on purpose.
super call requires the class to extend another class.